gebru/roster.json

```diff
@@ -189,6 +189,8 @@
       "materials": [
         {
             "type": "cv"
+            "type": "video",
+            "url": "https://youtu.be/pSXnMD5ug7Q"
```

Comment:
@akennedy205 
I know the name of the key is misleading, but can you please replace the full url with the video_id
i.e, "url": "pSXnMD5ug7Q"